cli/fuzzers/afl/register_afl_fuzzer.cpp

```diff
@@ -21,12 +21,15 @@
 #include "fuzzuf/executor/afl_executor_interface.hpp"
 #include <boost/program_options.hpp>
 
-namespace fuzzuf::algorithm::afl {
+namespace fuzzuf::cli::fuzzer::afl {
```

Comment:
FuzzerBuilderRegister can work no matter where variables of that type are declared.
So it is better if `global_afl_register` is in `fuzzuf::cli::fuzzer::afl`, considering `BuildAFLFuzzerFromArgs` is in namespace `fuzzuf::cli::fuzzer::afl`.
The same is true for other fuzzers.